internal/installcfg/installcfg_test.go

```diff
@@ -93,6 +94,29 @@ var _ = Describe("installcfg", func() {
 		Expect(result.Proxy.HTTPSProxy).Should(Equal(proxyURL))
 	})
 
+	It("correctly applies cluster overrides", func() {
+		var result InstallerConfigBaremetal
+		data, err := GetInstallConfig(logrus.New(), &cluster)
+		Expect(err).ShouldNot(HaveOccurred())
+		err = yaml.Unmarshal(data, &result)
+		Expect(err).ShouldNot(HaveOccurred())
+		// test that overrides worked
+		Expect(result.Networking.NetworkType).Should(Equal("OVN-Kubernetes"))
+		Expect(result.FIPS).Should(Equal(true))
+		// test that existing values are kept
+		Expect(result.APIVersion).Should(Equal("v1"))
+		Expect(result.BaseDomain).Should(Equal("redhat.com"))
+	})
+
+	It("doesn't fail with empty overrides", func() {
+		var result InstallerConfigBaremetal
+		cluster.InstallConfigOverrides = ""
+		data, err := GetInstallConfig(logrus.New(), &cluster)
+		Expect(err).ShouldNot(HaveOccurred())
+		err = yaml.Unmarshal(data, &result)
+		Expect(err).ShouldNot(HaveOccurred())
```

Comment:
Verify that Networking isn't set
```suggestion
		Expect(err).ShouldNot(HaveOccurred())
		Expect(result.Networking.NetworkType).Should(Equal(""))

```

Reply:
Actually in this case this would be set to `OpenShiftSDN` but I can make that assertion.